tests/views/test_response_correlation.py

```diff
@@ -118,16 +116,16 @@ def test_show_respo_with_obs(mock_data, dash_duo):
 
     select_ensemble(dash_duo, plugin, "default3")
 
-    resp_selector_id = plugin.uuid("parameter-selector-multi-resp")
-    default_responses = get_options(dash_duo, "#" + resp_selector_id)
-
     expected_def_responses = ["FGPT", "FOPR", "SNAKE_OIL_GPR_DIFF", "WOPR:OP1"]
-    assert default_responses == expected_def_responses
-
     expected_obs_responses = ["FOPR", "WOPR:OP1"]
+
+    resp_selector_id = "#" + plugin.uuid("parameter-selector-multi-resp")
+
+    dash_duo.wait_for_text_to_equal(resp_selector_id, "\n".join(expected_def_responses))
```

Comment:
Any reason to not having the timeout specification? `, timeout=5`

Reply:
What means `def` in `expected_def_responses`? If it is default, wouldn|t it be better to have ` default` instead of `def`?

Reply:
definitely good to write out default, not a lot of reason anymore nowadays to use abbreviated words, in my humble opinion ^^

i didn't see a reason to deviate from the default timeout (10 secons I guess?) in this particular, so i went with the default.

do you see a reason to deviate from the default? is it too long? : )

Reply:
did a couple of renames expanding abbreviated words.

Reply:
Timeout comment replied below. Nice :-)